src/certificates.ts

```diff
@@ -0,0 +1,16 @@
+
+import { ByteArray } from "@project-chip/matter.js";
```

Comment:
For consistency, this file name should start with a capital letter

Reply:
OK. @Apollon77 also suggested name change to DeveloperCertificates.